CHANGES-4.3.md

```diff
@@ -21,6 +21,7 @@ File format:
 ### Bug fixes
 * List subscription topic (/api/v4/subscriptions), the result do not match with multiple conditions.  
 
+* SSL closed error bug fixed
```

Comment:
```suggestion
* SSL closed error bug fixed for redis client.
```

Reply:
reason: In EMQX there are quite many different SSL connections, without stating which one had issue, this change log isn't really providing any value.